lib/gris/generators/templates/scaffold/config/application.rb.tt

```diff
@@ -3,8 +3,8 @@ require File.expand_path('../boot', __FILE__)
 # autoload initalizers
 Dir['./config/initializers/**/*.rb'].map { |file| require file }
 
-# autoload app
-relative_load_paths = %w(app/endpoints app/presenters app/models)
+# Autoload directories within /app
+relative_load_paths = Dir.glob 'app/**/*/'
```

Comment:
This can do some weird stuff if we have model subdirectories that don't correspond to the actual modules for those classes, such as

    /app/models/domain/*.rb

Just something to be aware of. I don't think it's a good idea to use subdirectories for models without namespacing them anyway.